docs/docsite/rst/galaxy/dev_guide.rst

```diff
@@ -95,32 +95,12 @@ Alternatively, the role_skeleton and ignoring of files can be configured via ans
 Authenticate with Galaxy
 ------------------------
 
-Using the ``import``, ``delete`` and ``setup`` commands to manage your roles on the Galaxy website requires authentication, and the ``login`` command
-can be used to do just that. Before you can use the ``login`` command, you must create an account on the Galaxy website.
+Using the ``import``, ``delete`` and ``setup`` commands to manage your roles on the Galaxy website requires authentication in the form of an API key, you must create an account on the Galaxy website.
 
-The ``login`` command requires using your GitHub credentials. You can use your username and password, or you can create a `personal access token <https://help.github.com/articles/creating-an-access-token-for-command-line-use/>`_. If you choose to create a token, grant minimal access to the token, as it is used just to verify identify.
+#. Log in to the Galaxy website and open the `Preferences <https://galaxy.ansible.com/me/preferences>`_ view.
+#. Select **Show API key** and then copy it.
 
-The following shows authenticating with the Galaxy website using a GitHub username and password:
-
-.. code-block:: text
-
-   $ ansible-galaxy login
-
-   We need your GitHub login to identify you.
-   This information will not be sent to Galaxy, only to api.github.com.
-   The password will not be displayed.
-
-   Use --github-token if you do not want to enter your password.
-
-   GitHub Username: dsmith
-   Password for dsmith:
-   Successfully logged into Galaxy as dsmith
-
-When you choose to use your username and password, your password is not sent to Galaxy. It is used to authenticates with GitHub and create a personal access token.
-It then sends the token to Galaxy, which in turn verifies that your identity and returns a Galaxy access token. After authentication completes the GitHub token is
-destroyed.
-
-If you do not want to use your GitHub password, or if you have two-factor authentication enabled with GitHub, use the ``--github-token`` option to pass a personal access token that you create.
+#. Save your token in the `.ansible/galaxy_token` file.
```

Comment:
This should reference the config option instead of a hardcoded filename https://github.com/ansible/ansible/blob/devel/lib/ansible/config/base.yml#L1397-L1404.